tests/functional/adapter/materialized_view_tests/test_materialized_views.py

```diff
@@ -233,3 +237,28 @@ class TestRedshiftMaterializedViewChangesFail(
 ):
     # Note: using retries doesn't work when we expect `dbt_run` to fail
     pass
+
+
+NO_BACKUP_MATERIALIZED_VIEW = """
+{{ config(
+    materialized='materialized_view',
+    backup=False
+) }}
+select * from {{ ref('my_seed') }}
+"""
+
+
+class TestRedshiftMaterializedViewWithBackupConfig:
+    @pytest.fixture(scope="class", autouse=True)
+    def models(self):
+        yield {
+            "my_materialized_view.sql": NO_BACKUP_MATERIALIZED_VIEW,
+        }
+
+    @pytest.fixture(scope="class", autouse=True)
+    def seeds(self):
+        return {"my_seed.csv": MY_SEED}
+
+    def test_running_mv_with_backup_false_succeeds(self, project):
```

Comment:
This was the best I could think of too. We can verify that it runs successfully, but we can't inspect the database to ensure that backup was turned off. I suppose we could look for the backup phrase in the logs as well? I only say that because removing the backup phrase from the create template entirely would also pass this test.

Reply:
I can double check that it's included in the node config